main.tf

```diff
@@ -53,7 +53,9 @@ module "consul_servers" {
   # WARNING! By specifying just the "family" name of the Image, Google will automatically use the latest Consul image.
   # In production, you should specify the exact image name to make it clear which image the current Consul servers are
   # deployed with.
-  source_image = "${var.consul_server_source_image}"
+  source_image_family = "${var.consul_server_source_image_family}"
```

Comment:
I believe you can just specify a family name for the image already, correct? Moreover, we still need to allow users to specify an exact image if they want. Given that, I'm not sure we need this change...